.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "editor.formatOnSave": true,
+}
```

Comment:
Should this be in the gitignore? I wasn't sure, so I've just left it here for now

Reply:
yes - please add to gitignore, I think we should keep IDE-specific files out of here

Reply:
nice - I've removed it now and update the gitignore